spec/models/firmware_registry/rest_api_depot_spec.rb

```diff
@@ -0,0 +1,175 @@
+describe FirmwareRegistry::RestApiDepot do
+  before do
+    VCR.configure do |config|
+      config.filter_sensitive_data('AUTHORIZATION') { Base64.encode64("#{user}:#{pass}").chomp }
+      config.before_record { |interaction| interaction.request.uri.downcase! }
+    end
+    # Is seems some other test is turning VCR off in some random cases.
+    # We're best off manually turning it on.
+    VCR.turn_on!
```

Comment:
Hm this is strange that it'd be needed, I'm okay with this for now but we should look at why it is needed